demisto_sdk/commands/openapi_codegen/openapi_codegen.py

```diff
@@ -0,0 +1,837 @@
+                    'properties': arg.get('properties', {})
+                })
+
+            for output in function['outputs']:
+                command['outputs'].append({
+                    'name': output['name'],
+                    'description': output['description'],
+                    'type': output['type']
+                })
+
+                if output['name'] in self.unique_keys:
+                    command['unique_key'] = output['name']
+
+            if 'unique_key' not in command:
+                command['unique_key'] = ''
+
+            configuration['commands'].append(command)
+
+        configuration['code_type'] = 'python'
+        configuration['code_subtype'] = 'python3'
+        configuration['docker_image'] = 'demisto/python3:3.8.3.9324'
+
+        self.configuration = configuration
+
+    def get_python_code(self):
+        # Use the code from baseCode in py as the basis
+        data = base_code
+
+        # Build the functions from configuration file
+        functions = []
+        req_functions = []
+        for command in self.configuration['commands']:
+            function, req_function = self.get_command_functions(command)
+            functions.append(function)
+            req_functions.append(req_function)
+
+        data = data.replace('$FUNCTIONS$', '\n'.join(functions))
+        data = data.replace('$BASEURL$', self.base_path)
+        client = base_client.replace('$REQUESTFUNCS$', '\n'.join(req_functions))
+        data = data.replace('$CLIENT$', client)
+
+        if API_KEY_AUTH_TYPE in self.configuration['auth']:
+            data = data.replace('$BEARERAUTHPARAMS$', base_token)
+        else:
+            data = data.replace('$BEARERAUTHPARAMS$', '')
+
+        if CREDENTIALS_AUTH_TYPE in self.configuration['auth']:
+            data = data.replace('$BASEAUTHPARAMS$', base_credentials)
+            data = data.replace('$BASEAUTH$', base_basic_auth)
+        else:
+            data = data.replace('$BASEAUTHPARAMS$', '')
+            data = data.replace('$BASEAUTH$', 'None')
+
+        list_functions = []
+
+        # Add the command mappings:
+        for command in self.functions:
+            prefix = ''
+            if self.command_prefix:
+                prefix = f'{self.command_prefix}-'
+
+            function = base_list_functions.replace(
+                '$FUNCTIONNAME$', f"{prefix}{command['name']}".replace('_', '-'))
+            fn = command['name'].replace('-', '_')
+            function = function.replace('$FUNCTIONCOMMAND$', f'{fn}_command')
+            list_functions.append(function)
+
+        data = data.replace('$COMMANDSLIST$', '\n\t'.join(list_functions))
+        self.print_with_verbose('Finished creating the python code.')
+
+        if self.fix_code:
+            self.print_with_verbose('Fixing the code with autopep8...')
+            data = autopep8.fix_code(data)
+
+        return data
+
+    def get_command_functions(self, command):

```

Comment:
the func name here is confusing, as it also returns the request functions
and that function is too long imo, consider splitting it